helm/cluster-apps-operator/templates/configmap.yaml

```diff
@@ -28,7 +28,7 @@ data:
             override: {{ toYaml .Values.release.app.config.override | indent 12 }}
       workload:
         cluster:
-          baseDomain: '{{ .Values.baseDomain }}'
+          baseDomain: '{{ .Values.base }}'
```

Comment:
The name changed so this is prep for integration with the config repo. 
https://github.com/giantswarm/config/blob/aeb433cb15f64159e09f0c4d26ca8714257dd951/installations/ginger/config.yaml.patch#L1